### ISSUE TITLE
Playground: do not cut values on the stack

### ISSUE THREAD

**Issue by elawbek** [comitylabs/evm.codes#179]
I think it would be more convenient if `PUSH1 0x42` would output in stack not just `42` but `0x0000...42`, i.e. with a size of 32 bytes.

If my suggestion does not suit you, let me know.

**Comment by dorlevi** [comment]
Personally, I like the current behavior WDYT? @thevaizman

**Comment by thevaizman** [comment]
From an aesthetic POV I'm with @dorlevi on this one, even though I understand @elawbek's point that the values shown do not replicate how they're really stored on the stack.
I personally do not see much value in "expanding" the stack values and I'd rather keep the view less cluttered and maybe just add a disclaimer/reminder next to the `STACK` header that all items are 32 bytes long.

Having said that, if there's a large demand from the community, we should go with them :)

**Comment by elawbek** [comment]
If add a flag to show or not show the full size of the stack value?

I don't know how to check if there is a demand from the community.

**Comment by elawbek** [comment]
As a frequent user, I am used to the fact that when 
```
PUSH1 0x42
PUSH1 0x00
MSTORE 
```
there is just the number 42 in the stack, but in the memory is shifted and the number is actually written not to the first byte but to the 32nd (I understand that by definition `MSTORE` writes 32 bytes into memory and this should be clear anyway), but I think that new users will have a better understanding of the 32-byte number in the stack and why it is written to memory this way.
Also, when learning how bit shifts work, it will be very handy to see how far to the left or right the number has shifted, without having to count in your mind the zeros that must be in front of the significant part of the value.

That's all. If you think these reasons aren't enough, let me know. 
Thank you.

**Comment by SovaSlava** [comment]
I agree with @elawbek  that its interesting feature and I'd like to see real view of variables in stack.

**Comment by Lucienest** [comment]
I agree with @elawbek there should be a way to toggle between aesthetic and actual view.
